ckan/tests/migration/revision_legacy_code_tests.py

```diff
@@ -0,0 +1,307 @@
+# encoding: utf-8
+
+# This file is at the top of the ckan repository, because it needs to be run
+# separately from all the other tests, because when it imports
+# revision_legacy_code.py it changes the core model, which causes a number of
+# test failures which we're not concerned about.
```

Comment:
You've created a duplicate of `test_revision_legacy_code.py`? I guess you're thinking about moving that test here? It would be a better home, but I don't see how you managed to get tests pass with it here, when we previously found it problematic for tests that followed on - as you noted [here](https://github.com/ckan/ckan/pull/4784#issuecomment-546941391). Have you fixed something or might this come back to bite us in future?

Reply:
Sorry, this was a mistake (it was lying around from previous tests I did on the vdm branch). I've removed it now